docs/experimental-features/user-risk-score.asciidoc

```diff
@@ -136,6 +140,67 @@ The data tables beneath the histogram display associated rules, users, and MITRE
+
+* `Saved object already exists`
+* `Transform already exists`
+* `Ingest pipeline already exists`
+
+In this case, we recommend that you manually delete the user risk score module, and then re-enable it. To manually delete the module:
+
+. Delete the user risk score saved objects:
+.. From the {kib} main menu, go to **Stack Management** -> **Kibana** -> **Saved Objects**.
+.. Delete the saved objects that have the `User Risk Score - <space-id>` tag.
++
+[role="screenshot"]
+image::images/delete-urs-saved-objects.png[Delete user risk score saved objects]
+.. Delete the `User Risk Score - <space-id>` tag.
++
+[role="screenshot"]
+image::images/delete-urs-tag.png[Delete user risk score tag]
+. Stop and delete the user risk score transforms. You can do this through the {kib} UI or using the {ref}/stop-transform.html[Stop transform API] and {ref}/delete-transform.html[Delete transform API].
+** To delete the user risk score transforms through the {kib} UI:
+.. From the {kib} main menu, go to **Stack Management** -> **Data** -> **Transforms**.
+.. Stop the following transforms, then delete them:
+*** `ml_userriskscore_latest_transform_<space-id>`
+*** `ml_userriskscore_pivot_transform_<space-id>`
+** To delete the user risk score transforms using the API, run the following commands in Console:
+.. To stop and delete the latest transform:
++
+[source,console]
+----------------------------------
+POST _transform/ml_userriskscore_latest_transform_<space-id>/_stop
+DELETE _transform/ml_userriskscore_latest_transform_<space-id>
+----------------------------------
+.. To stop and delete the pivot transform:
++
+[source,console]
+----------------------------------
+POST _transform/ml_userriskscore_pivot_transform_<space-id>/_stop
+DELETE _transform/ml_userriskscore_pivot_transform_<space-id>
+----------------------------------
+. Delete the user risk score ingest pipeline. You can do this through the {kib} UI or using the {ref}/delete-pipeline-api.html[Delete pipeline API].
+** To delete the user risk score ingest pipeline through the {kib} UI:
+.. From the {kib} main menu, go to **Stack Management** -> **Ingest** -> **Ingest Pipelines**.
+.. Delete the `ml_userriskscore_ingest_pipeline_<space-id>` ingest pipeline.
+** To delete the user risk score ingest pipeline using the Delete pipeline API, run the following command in Console:
++
+[source,console]
+----------------------------------
+DELETE /_ingest/pipeline/ml_userriskscore_ingest_pipeline_<space-id>
+----------------------------------
+. Delete the stored user risk score scripts using the {ref}/delete-stored-script-api.html[Delete stored script API]. In Console, run the following commands:
++
+[source,console]
+----------------------------------
+DELETE _scripts/ml_userriskscore_levels_script_<space-id>
+DELETE _scripts/ml_userriskscore_map_script_<space-id>
+DELETE _scripts/ml_userriskscore_reduce_script_<space-id>
+----------------------------------
 
```

Comment:
Perhaps this would be a good place to remind users to re-enable the modules, and link to the instructions for that.

Reply:
Makes sense! Updated in https://github.com/elastic/security-docs/pull/4074/commits/513b722bd492e5b1eadd703dca5a5f756234d834.